serverless/test/index.spec.ts

```diff
@@ -11,6 +11,7 @@ import {
 import { LogGroupDefinition } from "../src/forwarder";
 
 const LAMBDA_KEY = "HelloWorldFunction";
+const V_SEMVER_REGEX = /^v(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(-(0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(\.(0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*)?(\+[0-9a-zA-Z-]+(\.[0-9a-zA-Z-]+)*)?$/;
```

Comment:
What does `V_SEMVER` mean here? (I know semver refers to semantic version, but what's the "v" for?)

Reply:
`v` is the prefix in the regex

Reply:
Hmm, I would not have guessed that, I would just call this `VERSION_REGEX` or something, but up to you